### DISCUSSION TITLE
Pitch Bend from a MIDI Controller

### DISCUSSION THREAD

**Discussion by dasaxguy** [DISTRHO/Cardinal#592] [Q&A]
Hello all! I am using Cardinal in Ableton 10, controlled with an EWI Wind Synth/Controller and I am confused as to how to get pitch bend  to work. In VCV Rack standalone, and in any other VST it just works. 

Any pointers?

EDIT 1 - I am on windows 10 pro.

**Comment by dromer** [comment]
Hmm, works for me with Cardinal and CardinalSynth vst3 and clap in Bitwig.

I am on Linux, so maybe your OS? (you never said which)

**Reply by dasaxguy** [reply]
Hello, I am on windows 10.

**Reply by falkTX** [reply]
also works fine for me in JACK standalone.

I suggest to add the fundamental scope on the host midi pitchbend output and see if anything appears/changes

**Reply by dromer** [reply]
Yeah something like

![](https://i.imgur.com/2ZoGgk0.png)

**Reply by dasaxguy** [reply]
I'll Give this a go!
